data/projects/integrate-pyflakes-AST.md

```diff
@@ -0,0 +1,21 @@
+This project focuses on integrating the [`pyflakes`](https://pypi.python.org/pypi/pyflakes)-enhanced [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree) into coala and wrap flake8 AST plugins [`flake8-future-import`](https://github.com/xZise/flake8-future-import) and [`flake8-string-format`](https://github.com/xZise/flake8-string-format) using it.
```

Comment:
The problem we are trying to solve:

pycodestyle supports plugins running AST based checks u.
flake8 has many plugins which are commonly used in the python development community.  flake8 is a wrapper, similar to coala, that provides that use the python-enhanced AST to its plugins.  Instead of coala using the flake8 wrapper, which adds another level of complexity and uncertainty with regards to configuration, coala would like to directly load the python-enhanced AST and invoke the flake8 plugins.
This allows the coala and flake8 communities to cooperate on plugin development using common plugin framework.